tools/state-viewer/src/latest_witnesses.rs

```diff
@@ -57,3 +82,53 @@ impl LatestWitnessesCmd {
         }
     }
 }
+
+#[derive(Parser)]
+pub struct ValidateWitnessCmd {
+    /// File with state witness saved as vector in JSON.
+    #[arg(long)]
+    input_file: PathBuf,
+}
+
+impl ValidateWitnessCmd {
+    pub(crate) fn run(&self, home_dir: &Path, near_config: NearConfig, store: Store) {
+        let encoded_witness: Vec<u8> =
+            serde_json::from_reader(BufReader::new(std::fs::File::open(&self.input_file).unwrap()))
```

Comment:
I wonder if the serialization format is user-friendly.
The `--binary` just debug-prints a `Vec<u8>`, at the time I thought that it would make it easy to paste into code or something, but now on second thought I think it might be annoying. Pasting multiple megabytes of text into VSCode sounds like a bad idea.
Apart from that the command can print out multiple witnesses in multiple lines, along with some other information.

Maybe it would be better if the `--binary` flag outputted a serialized `Vec<ChunkStateWitness>` as raw non-text bytes? That could be saved to a file and read directly, without treating the array as json (although that's pretty clever!).

@Longarithm you actually used the tool in the wild to debug stuff, do you think the current format is usable? WDYT?